octopoes/octopoes/models/ooi/findings.py

```diff
@@ -73,6 +73,10 @@ class FindingType(OOI):
     @classmethod
     def format_reference_human_readable(cls, reference: Reference) -> str:
         return reference.tokenized.id
+    
+    @classmethod
+    def type_from_raw(cls, raw_input:dict):
+        return KATFindingType
```

Comment:
This should differentiate over the various findingtypes. Luckily, they are are prefixed with a usable prefix (eg, KAT-, or CVE-)

Reply:
It's changed. KATFindings is the default if any FindingType class cannot be assigned.